src/views/Releases/Release/index.jsx

```diff
@@ -4,7 +4,7 @@ import Dashboard from '../../../components/Dashboard';
 
 export default function Release() {
   return (
-    <Dashboard>
+    <Dashboard title="Release">
```

Comment:
For the specific-object pages like this, what do you think about adding or replacing this with the object name? Eg: `Release: Firefox-67.0-build1` or `Rule: firefox-release`, etc.

Reply:
Makes sense. I'll add that prior to merging.

Reply:
Actually, it's probably not needed since the first field of a view will usually tell the user what view they're looking at. For example, in http://localhost:9000/required-signoffs/Firefox, the first field shows the product as "Firefox". Having that value duplicated is probably unnecessary. I'll merge as is but we can probably consider adding it to the view later if there's confusion. Hope that's ok.